bluesky/plans.py

```diff
@@ -1790,6 +1809,9 @@ def relative_log_scan(detectors, motor, start, stop, num, *, per_step=None,
     `bluesky.plans.log_scan`
     """
     # TODO read initial positions (redundantly) so they can be put in md here
+    if md is None:
+        md = {}
+    md = ChainMap(md, {'plan_name': 'relative_log_scan'})
```

Comment:
It is; in an offline-conversion, @tacaswell established that the precedence order is the opposite of what he was thinking here.